src/Vsix/Merq.Vsix/Merq.Vsix.csproj

```diff
@@ -76,7 +80,64 @@
     <BindingRedirect Include="Merq" />
     <BindingRedirect Include="Merq.Core" />
   </ItemGroup>
-  
+
+  <!--
+    See https://github.com/xamarin/UITools/pull/705/commits/4a65bea18148f7ac2ecd34419450eda3550a776a by Thomas Sparks
+
+    TL;DR: ValidateVsixReferencedAssemblies locks the file, preventing Signing from succeeding
+  -->
+  <Target Name="TempRemoveVsixSourceItems" BeforeTargets="ValidateVsixReferencedAssemblies">
+    <ItemGroup>
+      <VSIXSourceItem Remove="$(IntermediateOutputPath)Merq.Vsix.dll"/>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="ReturnVsixSourceItems" AfterTargets="ValidateVsixReferencedAssemblies">
+    <ItemGroup>
+      <VSIXSourceItem Include="$(IntermediateOutputPath)Merq.Vsix.dll"/>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="GetFilesToSign" BeforeTargets="SignVsixContentFiles" Condition="'$(Configuration)' == 'Release'">
+    <ItemGroup>
+      <FilesToSign Include="$(IntermediateOutputPath)Merq.Vsix.dll">
+        <Authenticode>Microsoft400</Authenticode>
+      </FilesToSign>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="CopyNuGetFilesToOutputDir" BeforeTargets="SignVsix" AfterTargets="CreateVsixContainer" Condition="'$(Configuration)' == 'Release'">
+    <ItemGroup>
+      <NupkgFiles Include="$(PackageOutputPath)\Merq.*.nupkg" />
+    </ItemGroup>
+    <Copy
+      SourceFiles="@(NupkgFiles)"
+      DestinationFolder="$(OutDir)"
+    />
+  </Target>
+
+  <Target Name="SignVsix" BeforeTargets="SignFiles" DependsOnTargets="GetFilesToSign;CreateVsixContainer;CopyNuGetFilesToOutputDir" Condition="'$(Configuration)' == 'Release'">
+    <ItemGroup>
+      <FilesToSign Include="$(OutDir)$(TargetVsixContainerName)">
+        <Authenticode>VsixSHA2</Authenticode>
+      </FilesToSign>
+      <FilesToSign Include="$(OutDir)Merq.*.nupkg">
+        <Authenticode>NuGet</Authenticode>
+      </FilesToSign>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="CopySignedNuGetFilesToOutputDir" AfterTargets="SignFiles" Condition="'$(Configuration)' == 'Release'">
```

Comment:
More accurate target name would probably be `...ToPackageDir`

Reply:
Resolved in [a1638c2](https://github.com/xamarin/Merq/pull/4/commits/a1638c2700063c6351a65c4d452b18c47d4b7b26)